api/matlab/arrus/BModeDisplay.m

```diff
@@ -3,38 +3,124 @@
     %
     % Currently is implemented as a simple MATLAB figure with dynamically
     % updated content.
-    %
+    % 
+    % :param reconstructionObject: object of class "Reconstruction" \
+    %   (obligatory input unless old syntax is used)
+    % :param dynamicRange: two-element vector [min, max], dynamic range \
+    %   limits to apply (optional name-value argument)
+    % :param cineLoopLength: positive scalar, number of frames stored in \
+    %   cineloop (optional name-value argument)
+    % :param persistence: persistence filtration weights (vector) or \
+    %   length (scalar) (optional name-value argument)
+    % :param bmodeTgc: applies linear TGC equal to bmodeTgc[dB]/depth \
```

Comment:
depth units should be given in docstring

Reply:
corrected: 
    % :param bmodeTgc: applies linear TGC from 0 at top to bmodeTgc[dB] \
    %   at bottom of the image (scalar) (optional name-value argument)